lib/fs.js

```diff
@@ -122,6 +142,10 @@ function nullCheck(path, callback) {
   return true;
 }
 
+function isFd(path) {
```

Comment:
That's just reverting the commits from #7168 so test-npm can be run. Not part of the PR.